### ISSUE TITLE
MSApp.createNewView is missing

### ISSUE THREAD

**Issue by RReverser** [SaschaNaz/uwp-type-generator#6]
As far as I understand, it's available in Windows 10, so should likely be included in `winrt-uwp`, but for some reason is not generated.

MSDN page: https://msdn.microsoft.com/library/windows/apps/dn252339

**Comment by saschanaz** [comment]
Currently MSApp is covered by lib.d.ts, not winrt.d.ts. BTW, you should use `window.open` instead of `MSApp.createNewView`: https://social.msdn.microsoft.com/Forums/windowsapps/en-US/d800a2bb-0ca3-4cd0-9631-1d2ef66babe6/uwphtml-porting-code-from-the-multiple-views-sample-is-crashing?forum=wpdevelop

**Comment by RReverser** [comment]
Oh, I see, so apparently this method was removed in Win10, but docs and samples don't reflect this change yet. That explains why it's missing in `lib.d.ts` then.

As for `window.open` - that was my first idea which I wanted to use, but [docs](https://msdn.microsoft.com/en-us/library/ms535873%28v=vs.85%29.aspx) about using it in app context sound somewhat scary:

> Navigates the app window to the specified location. 

(sounds like it navigates away from the main view and not just creates a new one)

Now that I tried, it seems to work fine in both desktop & tablet modes.
